pkg/minikube/bootstrapper/images/images.go

```diff
@@ -132,7 +141,38 @@ func CachedImages(imageRepository string, kubernetesVersionStr string) (string,
 		minikubeRepository + "storage-provisioner" + ArchTag(false) + "v1.8.1",
 	}...)
 
-	return podInfraContainerImage, images
+	return images
+}
+
+func PauseImage(imageRepository string, kubernetesVersionStr string) string {
+	imageRepository = getImageRepository(imageRepository)
+
+	v1_16plus := semver.MustParseRange(">=1.16.0")
+	v1_14plus := semver.MustParseRange(">=1.14.0 <1.16.0")
+	v1_13 := semver.MustParseRange(">=1.13.0 <1.14.0")
+	v1_12 := semver.MustParseRange(">=1.12.0 <1.13.0")
+	v1_11 := semver.MustParseRange(">=1.11.0 <1.12.0")
+
+	kubernetesVersion, err := semver.Make(strings.TrimPrefix(kubernetesVersionStr, minikubeVersion.VersionPrefix))
+	if err != nil {
+		glog.Errorln("Error parsing version semver: ", err)
+	}
+
+	var podInfraContainerImage string
+	if v1_16plus(kubernetesVersion) {
```

Comment:
I think here it is much better to use switch?

Reply:
Sounds good & Done.